msbuild/Xamarin.iOS.Tasks.Windows/Xamarin.iOS.HotRestart.targets

```diff
@@ -247,7 +257,7 @@
 		Condition="'$(_CanOutputAppBundle)' == 'true' And '$(IsAppExtension)' == 'false' And '$(IsHotRestartBuild)' == 'true' And '$(EnableCodeSigning)' != 'false'"
 		DependsOnTargets="_CreateHotRestartCachedBundle;_CollectCodeSignHotRestartInputs"
 		Inputs="@(_CodeSignHotRestartInputs)"
-		Outputs="$(HotRestartAppBundlePath)\_CodeSignature\CodeResources">
+		Outputs="@(_CodeSignHotRestartInputs -> '%(Outputs)')">
```

Comment:
Does this mean that code signing does _not_ create the `_CodeSignature\CodeResources` file in the app bundle?

Reply:
> Does this mean that code signing does _not_ create the `_CodeSignature\CodeResources` file in the app bundle?

Yes, it creates it. But for some reason it always shows me this message and never satisfies de inputs/outputs statement:

Input file "bin\Debug\net8.0-ios\ios-arm64\MauiApp35.app\Info.plist" is newer than output file "C:\Users\maagno\AppData\Local\Temp\Xamarin\HotRestart\Bundles\1.0.103.11\4fdcd990\MauiApp35.app\_CodeSignature\CodeResources".

Reply:
Maybe an earlier task always writes Info.plist?